.doc_gen/metadata/polly_metadata.yaml

```diff
@@ -0,0 +1,65 @@
+# zexi 0.4.0
+polly_DescribeVoices:
+  title: Return the list of &POL; voices that are available for use when requesting speech synthesis using an &AWS; SDK
+  title_abbrev: Get available voices
+  synopsis: get the available &POL; voices.
```

Comment:
Move extra info out of title into synopsis.

Reply:
I've pared down the titles for all of the Polly code examples.